keps/sig-node/2837-pod-level-resource-spec/README.md

```diff
@@ -1576,23 +1298,29 @@ feature gate and by setting the new `resources` fields in PodSpec at Pod level.
 * Documentation mentioning high level design.
 
 
-#### Phase 2:  Beta (target 1.33)
-
+#### Phase 2:  Beta (target 1.34)
 
 * User Feedback.
 * Feature is disabled by default. It is an opt-in feature which can be enabled by setting the new fields in PodSpec.
 * Support for all features that are scoped out of alpha phase in the KEP design.
 * Unit test coverage for additional features supported in Beta.
 * E2E tests for additional features supported in Beta.
 * Documentation update and blog post to announce feature in beta phase.
-* [Tentative] Benchmark tests for resource usage with and without pod level resources for bursty workloads.
+* Benchmark tests for resource usage with and without pod level resources for bursty workloads.
   * Use kube_pod_container_resource_usage metric to check resource utilization.
-* [TBD] In-place pod resize support either as a part of this KEP or a separate KEP/feature.
+* In-place pod resize support guarded behind a separate feature gate -
+  `InPlacePodLevelResourcesVerticalScaling` i.e. (Issue#5419)[https://github.com/kubernetes/enhancements/issues/5419]
+* Suggest Karpenter folks to use pod resource requests from k8s.io/component-helper
+  or update https://github.com/kubernetes-sigs/karpenter/blob/v1.5.0/pkg/utils/resources/resources.go#L111-L144
 
 #### GA (stable)
 
-* TBD
-
+* VPA integration of feature moved to beta.
+* No major bugs reported for 3 months.
+* Pod Level Resources Support With In Place Pod Vertical Scaling KEP is past alpha.
+* User feedback (ideally from at least two distinct users) is green
+* Resource Allocation Managers i.e. Topology, Memory and CPU managers support with
+  Pod-level resources is past alpha.
 
 ### Upgrade / Downgrade Strategy
```

Comment:
1. In `###### Are there any tests for feature enablement/disablement?` you're describing tests that were planned for alpha, can you add there links to those tests? 
2. `###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?` is missing answers, it currently mentions only that it will be tested? Was it? Can you describe the steps? 
3. In `###### Will enabling / using this feature result in increasing size or count of the existing API objects?` you're missing estimated size increase, and amount of new objects (see comments above that section)

Reply:
Addressed the concerns.